tideways/provision.sh

```diff
@@ -99,7 +99,7 @@ function enable_tideways_by_site() {
 }
 
 . "${DIR}/../mongodb/provision.sh"
-
+DIR=$(dirname "$0")
```

Comment:
```suggestion
# Set DIR back to undo the DIR set in the MongoDB provisioner
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

This should use the DIR command at the top, and have an explainer so it's clear why this is in use